src/service/api.tsx

```diff
@@ -19,35 +19,43 @@ const api = {
   getStore() {
     const serviceStore: any = localStorage.getItem(apiService.storeKey);
     serviceStore !== null
-      ? apiService.store = JSON.parse(serviceStore)
+      ? (apiService.store = JSON.parse(serviceStore))
       : this.setStore();
     return apiService.store;
   },
 
-  setStore(data: Object = {
-    history: [],
-    favorites: [],
-    theme: 1
-  }) {
+  setStore(
+    data: Object = {
+      history: [],
+      favorites: [],
+      theme: 1
+    }
+  ) {
     localStorage.setItem(apiService.storeKey, JSON.stringify(data));
   },
 
   switchTheme() {
     const store = this.getStore();
-    store.theme = (store.theme === Theme.light) ? Theme.dark : Theme.light;
+    store.theme = store.theme === Theme.light ? Theme.dark : Theme.light;
     this.setStore(store);
     return store.theme;
   },
 
   setFavoritesId(id: number) {
     const store = this.getStore();
-    const isThereAnId = store.favorites.includes(id);
-    if (isThereAnId) {
-      store.favorites.push(id)
-    };
+    const isNewId = store.favorites.includes(id);
+    if (!isNewId) {
+      store.favorites.push(id);
+    }
     this.setStore(store);
   },
 
+  isIdInFavorites(id: number) {
+    const store = api.getFavoritsIdList();
+    const isNewId = store.includes(id);
+    return !isNewId;
```

Comment:
why not just return !store.favorites.includes(id)

Reply:
fixed